app/scripts/lib/createMetaRPCHandler.test.js

```diff
@@ -70,7 +70,8 @@ describe('createMetaRPCHandler', () => {
       params: ['bar'],
     });
     streamTest.on('data', (data) => {
-      expect(data.error.message).toStrictEqual('foo-error');
+      expect(data.error.message).toStrictEqual('Internal JSON-RPC error.');
+      expect(data.error.data.cause.message).toStrictEqual('foo-error');
```

Comment:
As the `data` here is serialized JSON (and not an instance of any error object), it should still keep the `data.cause` and not have a `cause`.